Sources/TimeStorage.swift

```diff
@@ -55,7 +55,9 @@ public struct TimeStorage {
         case .standard:
             self.userDefaults = .standard
         case .appGroup(let groupName):
-            self.userDefaults = UserDefaults(suiteName: groupName) ?? .standard
+            let sharedDefaults = UserDefaults(suiteName: groupName)
+            assert(sharedDefaults != nil, "Could not create UserDefaults for group: '\(groupName)'")
```

Comment:
I wonder if a cleaner api would be to just update this case to take in a `UserDefaults` instance rather than a string. Then you don't even get into a state where you have this run time check in this library and it is up to clients to pass in what you expect

Reply:
I think the right abstraction here would be to make TimeStorage a protocol and let users of `Clock` set whatever `TimeStorage` they want, could be off a DB, userdefaults, memory or whatever.

Reply:
Note `Clock.storage` is public so it'd be a simple change